document/paragraph-rtl/main.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 FoxyUtils ehf. All rights reserved.
```

Comment:
should be "UniDoc ehf."  Maybe just skip the year from the text, as always needs update.

Would be good to have some description of what this example does etc

Reply:
Updated at c18e50b97f2ccb776fbc8ec080052eb1e1c7eb8c